src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -74,6 +74,96 @@ export class AttributesController extends BaseController {
         return this.ok(result);
     }
 
+    @GET
+    @Path("/Own/Repository")
+    @Accept("application/json")
+    public async getOwnRepositoryAttributes(@Context context: ServiceContext): Promise<Envelope> {
+        const onlyLatestVersionsQuery = context.request.query.onlyLatestVersions;
+        const onlyLatestVersions: boolean | undefined = typeof onlyLatestVersionsQuery === "string" ? onlyLatestVersionsQuery === "true" : false;
+        const result = await this.consumptionServices.attributes.getRepositoryAttributes({
+            onlyLatestVersions
+        });
```

Comment:
you are missing to add the rest of the query to query

Reply:
this actually has no additional query

Reply:
that's not good, we should add a query in the runtime to make it available